tutorials/io/30_reading_fnirs_data.py

```diff
@@ -241,15 +242,12 @@
 # The ficiduals are marked in blue, green and red.
 # See :ref:`plot_source_alignment` for more details.
 
-subjects_dir = mne.datasets.sample.data_path() + '/subjects'
+subjects_dir = op.join(mne.datasets.sample.data_path(), 'subjects')
 mne.datasets.fetch_fsaverage(subjects_dir=subjects_dir)
 
-trans = mne.channels.compute_native_head_t(montage)
-
-fig = mne.viz.create_3d_figure(size=(800, 600), bgcolor='white')
-fig = mne.viz.plot_alignment(
-    raw.info, trans=trans, subject='fsaverage', subjects_dir=subjects_dir,
-    surfaces=['brain', 'head'], coord_frame='mri', dig=True, show_axes=True,
-    fnirs=['channels', 'pairs', 'sources', 'detectors'], fig=fig)
-mne.viz.set_3d_view(figure=fig, azimuth=90, elevation=90, distance=0.5,
-                    focalpoint=(0., -0.01, 0.02))
+brain = mne.viz.Brain('fsaverage', subjects_dir=subjects_dir,
+                      alpha=0.5, cortex='low_contrast')
+brain.add_head()
+brain.add_sensors(raw)
+brain.enable_depth_peeling()
```

Comment:
I understand this is from another PR, but I am just curious. Why is depth peeling required here, but not in the other example? I looked at the doc string and it doesn't explain what depth peeling is or in what situations you should use it. So I looked up wikipedia for an explanation. But I am still missing an understanding of in which situations you should enable this option.

Reply:
It has to do with the rendering and you should probably ask Eric to get a better explanation of technically what is going on, but my understanding and from working through examples what I've seen is that if there are many transparent surfaces overlaid, you really want depth peeling but, if not, it can cause some funky effects like the MEG sensors in this PR.

Reply:
Thanks Alex, you've been a real gem teaching me along side these PRs.